### PR TITLE
Azure terraform example for use with launchpad

### PR DESCRIPTION
**Pull request by stevejr**
Fixes: #ENGORC-7646

Signed-off-by: Steve Richards <srichards@mirantis.com>

### CODE REVIEW

File: examples/terraform/azure/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Bootstrapping UCP cluster on Azure
+
+This directory provides an example flow for using Mirantis Launchpad with Terraform and Azur.
```
Comment (kke) [inline examples/terraform/azure/README.md]:
```suggestion
This directory provides an example flow for using Mirantis Launchpad with Terraform and Azure.
```

Comment (stevejr) [reply examples/terraform/azure/README.md]: Done

**Comment by stevejr** [comment]
@kke - Please hold on merging this PR as we are just seeing if all the Windows aspects work - currently seeing a DNS issue with kubernetes so want to make sure its not related to how the infra is provisioned.

**Comment by stevejr** [comment]
@kke - I think we are good with this PR.

**Pull request merged**